### ISSUE TITLE
Support for modifying labels

### ISSUE THREAD

**Issue by twaugh** [DBuildService/dockerfile-parse#2]
With the `structure` property it is quite easy to modify most values, by first finding the value to modify, then replacing the lines (known from `startline` and `endline`) in the `lines` property.

For example, for CMD:
```
for insn in parser.structure:
  if insn['instruction'] == 'CMD':
    content = 'CMD my-new-cmd-value\n'
    lines = parser.lines
    del lines[insn['startline']:insn['endline'] + 1]
    lines.insert(startline, content)
    parser.lines = lines
    break
```

However, for the LABEL instruction it is much more complicated as firstly there are two forms of LABEL, secondly there is quoting to deal with, and thirdly a single LABEL instruction can contain many labels.

It would be nice to provide support for modifying a single label, handling all these cases.

**Comment by jpopelka** [comment]
@twaugh  Please check #4, the new code might be familiar to you ;-)

**Comment by twaugh** [comment]
Closed via #4.
